source/localizable/models/relationships.md

```diff
@@ -224,6 +224,71 @@ this.get('store').findRecord('user', 1).then(function(user) {
 });
 ```
 
+### Retrieving Related Records
+
+When you request data from the server for a model that has relationships
+with one or more others, you may want to retrieve records corresponding to
+those related models at the same time.
+For example, when retrieving a blog post, you may need to access the comments
+associated with the post as well.
+The [JSON API specification allows](http://jsonapi.org/format/#fetching-includes)
+servers to accept a query parameter with the key `include` as a request to
+include those related records in the response returned to the client.
+The value of the parameter should be a comma-separated list of names of the
+relationships required.
+
+If you are using an adapter that supports JSON API, such as Ember's default
+[`JSONAPIAdapter`](http://emberjs.com/api/data/classes/DS.JSONAPIAdapter.html),
+you can easily add the `include` parameter to the server requests created by
+the `findRecord()`, `findAll()`, `query()` and `queryRecord()` methods.
+
+`findRecord()` and `findAll()` each take an `options` argument in which you can
+specify the `include` parameter.
+For example, given a `post` model that has a `hasMany` relationship with a `comment`
+model, when retrieving a specific post we can have the server also return that
+post's comments as follows:
+
+```app/routes/post.js
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params) {
+   return this.store.findRecord('post', params.post_id, {include: 'comments'});
+  }
+});
+```
+The post's comments would then be available in your template as `model.comments`.
+
+Nested relationships can be specified in the `include` parameter as a dot-separated
+sequence of relationship names. So to request both the post's comments and the
+authors of those comments the request would look like this:
+
+```app/routes/post.js
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params) {
```

Comment:
Will do. I was following examples in the API docs which seem to use the longer form.